client/src/App.js

```diff
@@ -3,12 +3,42 @@ import ViewEntries from './ViewEntries';
 import './reset.css';
 import './layout.css';
 import './styles.css';
+import ViewCreateEnty from './ViewCreateEntry';
+import { useState } from 'react';
+/* exported data */
+
+let data = {
```

Comment:
I think this needs to be state instead of a global variable. And maybe several different state variables, such as `view`, `editing`, and `data` (that is, `{entries, nextEntryId}`.

This will make it easier to change the view and track what is currently being edited.